core/ir/opnd_shared.c

```diff
@@ -2531,10 +2531,16 @@ reg_get_size(reg_id_t reg)
         return OPSZ_SCALABLE;
     if (reg >= DR_REG_P0 && reg <= DR_REG_P15)
         return OPSZ_SCALABLE_PRED;
+    if (reg == DR_REG_CNTVCT_EL0)
+        return OPSZ_8;
+    if (reg >= DR_REG_NZCV && reg <= DR_REG_FPSR)
+        return OPSZ_8;
```

Comment:
Probably better to ask @AssadHashmi or another AArch64 expert for a review rather than me -- @AssadHashmi if you could confirm that these status registers are 64-bit despite having only a few fields?

Reply:
@derekbruening @JerryYouxin: yes, they're 64 bits and `INSTR_CREATE_mrs()`/`INSTR_CREATE_msr()` are used to access the specified user-space (EL0) fields.

Reply:
I found these register size info from ARM Developer website: [CNTVCT_EL0](https://developer.arm.com/documentation/ddi0595/2020-12/AArch64-Registers/CNTVCT-EL0--Counter-timer-Virtual-Count-register?lang=en), [NZCV](https://developer.arm.com/documentation/ddi0595/2020-12/AArch64-Registers/NZCV--Condition-Flags?lang=en), [FPCR](https://developer.arm.com/documentation/ddi0595/2020-12/AArch64-Registers/FPCR--Floating-point-Control-Register?lang=en), [FPSR](https://developer.arm.com/documentation/ddi0595/2020-12/AArch64-Registers/FPSR--Floating-point-Status-Register?lang=en)